dev/io.openliberty.restfulWS.3.1_fat/fat/src/io/openliberty/restfulWS31/fat/SeBootstrapTest.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.restfulWS31.fat;
+
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import io.openliberty.restfulWS31.fat.sebootstrap.SebootstrapTestServlet;
+
+@RunWith(FATRunner.class)
+public class SeBootstrapTest extends FATServletClient {
+    private static final String war = "sebootstrap";
+
```

Comment:
Why are you doing a server at all?  It is Java SE bootstrap.  This seems like it could be a unit test in io.openliberty.org.jboss.resteasy.common.ee10 or a junit test in this FAT that doesn't start a server at all since the point is that it is supposed to be SE bootstrap.  No server needed.  Unless you are testing to get expected results when you try to call it inside of a server runtime which is supposed to get particular errors as well.  If that is the case you still need a Java SE test.  This one is a Java EE test that is calling JavaSE and should get IllegalStateException I think if I was reading the code correctly.

Reply:
Discussed with Jared.   It is optional (but valid) to test via container.   But I will add a Java SE variation.

Reply:
Unit test added.